.github/workflows/hatch_ci.yml

```diff
@@ -167,25 +172,34 @@ jobs:
 
       - name: Build
         shell: bash
-        run: hatch build
-
+        run: |
+          hatch build
+          hatch dep show requirements > requirements.txt
+          
       - name: Verify Packages
         run: |
           ls dist/*.tar.gz | xargs -n1 hatch run python -m twine check
           ls dist/*.whl | xargs -n1 hatch run python -m twine check
         shell: bash
+      
+      - name: Install Virtualenv for Whitesource Scan
```

Comment:
Required for whitesource scan. Hatch is not natively supported. So we rely on pip.